contrib/admission-tracer/manifests/deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       - command:
         - /usr/bin/admission-tracer
         image: quay.io/sjenning/admission-tracer:latest
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
         name: admission-tracer
```

Comment:
@sjenning Do we want to keep this as `Always`? I see that the image references the `latest` tag.

Reply:
This is a dev debugging tool, I think Always makes more sense